### PR TITLE
Add 2017 Election statistics

### PR DESCRIPTION
**Pull request by TinaMurimi**
#### What does this PR do?
- Add KPMG report elections statistics
- Add 2017 Election statistics

#### Description of Task to be completed?
Adding `2017 election statistics to DB`. Since 2015 data could not be overwritten, a new table was created to hold the 2017 data. Additionally, the `2017 Voter Registration tab` was added to the Hurmap KE website.

#### Screenshots
screen shot 2017-08-02 at 12 10 41

### CODE REVIEW

**Comment by DavidLemayian** [comment]
@tinamorale Looks really good. Let's start by resolving the conflicts.

File: hurumap_ke/tables.py [hunk 1]
```diff
@@ -32,7 +32,8 @@
 FieldTable(['crimereport'], table_per_level=False, year = '2015')
 FieldTable(['typesofcrime'], table_per_level=False, year = '2015')
 FieldTable(['healthratios'], table_per_level=False, year = '2014')
-FieldTable(['voterregistration'], table_per_level=False, year = '2014')
+FieldTable(['voterregistration_2015'], table_per_level=False, year = '2014')
```
Comment (DavidLemayian) [inline hurumap_ke/tables.py:35]: Why 2015?

**Review by DavidLemayian — COMMENTED** [review]
I really like the code clean up. Just one question left.

File: hurumap_ke/tables.py (hunk 1 above)
Comment (TinaMurimi) [reply hurumap_ke/tables.py:35]: In the previous data, it has "March 2015 and as at October 2015" and "December 2015"

**Pull request merged**